indexer/feature.cpp

```diff
@@ -375,6 +376,15 @@ void FeatureType::ParseMetadata() const
   m_metadataParsed = true;
 }
 
+void FeatureType::ParseAltitude() const
+{
+  if (m_altitudeParsed)
+    return;
+
+  m_pLoader->ParseAltitude();
```

Comment:
Please, take a look at RankTable - how it's implemented and how it's used. It does not depend on a FeatureType completely. I thought the same you should implement for heights.